app/Http/Controllers/API/v1/AuthController.php

```diff
@@ -52,34 +51,27 @@ public function register(Request $request): JsonResponse {
 
     /**
      * Login
+     *
      * @param Request $request
+     *
      * @return JsonResponse
      * @api v1
      */
     public function login(Request $request): JsonResponse {
-        $validator = Validator::make($request->all(), [
-            'email'    => ['required', 'email'],
-            'password' => ['required']
-        ]);
+        $validated = $request->validate(['login' => ['required'], 'password' => ['required']]);
```

Comment:
Maybe we can add a `max:255` to login. E-Mails are limited to 255 and usernames to 25.

Reply:
and `min:8` for password would be nice, too.

Reply:
Idk, if that's really necessary since it'll return false either way. 😅